src/app/sidebar/sidebar.component.html

```diff
@@ -134,3 +142,7 @@
         </ul>
     </ng-container>
 </div>
+
+<ng-template #ToolTipNotVerified>
+    <div>Profile not verified. Go to User Profile screen to finish onboarding process.</div>
+</ng-template>
```

Comment:
There are 3 possible states where buttons are being disabled:

1. User not verified (Identyum),
2. Wallet not initialized (Arkane),
3. Wallet not activated by admin.

Look at user reminder component for all cases.